### PR TITLE
data: profile addition by DARK DEVIL

### PR DESCRIPTION
**Pull request by nonamehtml**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -303,6 +303,20 @@ <h3 class="name">Rémi JARA</h3>
                 <a href="https://www.linkedin.com/in/remijara/" target="_blank"><i class="fa-brands fa-linkedin-in"></i></a>
             </div>
         </div>
+        
+                <div class="profile">
+            <div class="pfp"><img src="https://telegra.ph/file/8a257ffcbce41b06c315f.jpg" alt="User Image"></div>
```
Comment (oyepriyansh) [inline index.html:308]: the image link is invalid

Comment (oyepriyansh) [inline index.html:308]: the image link is invalid

**Review by oyepriyansh — CHANGES_REQUESTED** [review]

**Review by nonamehtml — COMMENTED** [review]
How can I get The github Img url🙃

File: index.html (hunk 1 above)
Comment (nonamehtml) [reply index.html:308]: Please Can you dm me on dc

I don't know how to get the Github Img link

**Review by oyepriyansh — APPROVED** [review]
LGTM

**Pull request merged**